Platon/backend/app/auth_system/views.py

```diff
@@ -256,8 +297,18 @@ def put(user_id, self):
                     # Tries to update account information of the user.
                     # If it fails, an error is raised.
                     try:
-                        # DO NOT FORGET TO WRITE CODE FOR PROFILE INFORMATION
+                        # Checks whether the inputted job already exists in the database,
```

Comment:
Please Also make the required changes for ResearchGate and Google Scholar Research information fetch